manifest_staging/charts/secrets-store-csi-driver/templates/podsecuritypolicy.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: secrets-store-csi-driver-psp
```

Comment:
If there is no privileged psp, wouldn't the pod creation request fail after the `restricted` psp is evaluated first?

Reply:
If one enables the PSP it is assumed that the cluster uses them and therefore at least one unprivileged policy should be there. Cert-Manager also uses this approach.
https://github.com/jetstack/cert-manager/blob/master/deploy/charts/cert-manager/templates/psp.yaml